lib/pronto/cli.rb

```diff
@@ -56,16 +58,19 @@ def run(path = nil)
       commit_options = %i[staged unstaged index]
       commit = commit_options.find { |o| options[o] } || options[:commit]
 
-      repo_workdir = ::Rugged::Repository.discover('.').workdir
+      repo_workdir = ::Rugged::Repository.discover(path).workdir
+      relative     = path.sub(repo_workdir, '')
+
       messages = Dir.chdir(repo_workdir) do
-        ::Pronto.run(commit, '.', formatters, path)
+        file = relative.length != path.length ? relative : nil
+        ::Pronto.run(commit, '.', formatters, file)
       end
       if options[:'exit-code']
         error_messages_count = messages.count { |m| m.level != :info }
         exit(error_messages_count)
       end
     rescue Rugged::RepositoryError
-      puts '"pronto" should be run from a git repository'
+      puts '"pronto" should be run on/from a git repository'
```

Comment:
"on/from" seems weird to me, but I know what you mean.

Consider these more succinct or verbose options:

>  "pronto" should target a git repository

> "pronto" must be run from within a git repository or must be supplied the path to a git repository